librosa/feature/spectral.py

```diff
@@ -1163,6 +1164,104 @@ def logfsgram(y=None, sr=22050, S=None, n_fft=4096,
     return cq_basis.dot(S)
 
 
+@cache
+def tonnetz(y=None, sr=22050, chromagram=None, norm=np.inf):
+    '''Computes the tonal centroid features (tonnetz).
+
+    Parameters
+    ----------
+    y : np.ndarray [shape=(n,)] or None
+        Audio time series.
+
+    sr : int > 0 [scalar]
+        sampling rate of `y`
+
+    chromagram : np.ndarray [shape=(n_chroma, t)] or None
+        Normalized energy for each chroma bin at each frame.
+
+        If `None`, a cqt chromagram is performed.
+
+    norm : float or None
+        Column-wise normalization.
+        See `librosa.util.normalize` for details.
+
+        If `None`, no normalization is performed.
+
+    Returns
+    -------
+    ton : np.ndarray [shape(6, t)]
+        Tonal centroid features for each frame.
+        Tonnetz dimensions:
+            0: Fifth x-axis
+            1: Fifth y-axis
+            2: Minor x-axis
+            3: Minor y-axis
+            4: Major x-axis
+            5: Major y-axis
+
+    See Also
+    --------
+    librosa.feature.chroma_cqt
```

Comment:
According to numpydoc style guidelines, function refs within the same submodule should be left bare, that is,

```
chroma_cqt
```
instead of
```
librosa.feature.chroma_cqt
```

This makes it easier for the hyperlinks at doc build.

(References into other submodules are fine, but you discard the package name.  So ```util.normalize``` instead of ```librosa.util.normalize```.)